tests/a_lambda/hooks.py

```diff
@@ -2,5 +2,5 @@
 
 
 @hooks.before("/alambda > GET > 500 > application/json")
-def before_alambda_put_500(transaction):
```

Comment:
would be nice to have  a_lambda as a name, alambda is a little bit weird :)
What's your thoughts?

Reply:
fixed!